terraform/modules/lambda/iam.tf

```diff
@@ -1,11 +1,15 @@
+locals {
+  managed_policy_arns = concat([aws_iam_policy.lambda.arn], var.policy_arns)
+}
+
 resource "aws_iam_role" "lambda" {
-  assume_role_policy = data.aws_iam_policy_document.lambda_assume_role.json
-  name_prefix        = substr(var.lambda_function_name, 0, 38)
+  assume_role_policy  = data.aws_iam_policy_document.lambda_assume_role.json
+  name_prefix         = substr(var.lambda_function_name, 0, 38)
```

Comment:
Bit indirect - worth a comment to show what this should result in?

Reply:
I am not sure it needs one because it is `name_prefix = substr(var.lambda_function_name, 0, 38)` pattern is a common one for ensuring uniquely named resources especially where it is desirable to create the replacement resource before deleting the current one, and duplicate names are not allowed.